seedServer/src/main/java/rest/Register.java

```diff
@@ -0,0 +1,50 @@
+package rest;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.nimbusds.jose.JOSEException;
+import facades.UserFacade;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import security.PasswordStorage;
+
+@Path("register")
+public class Register {
+
+    Gson gson;
+
+    public Register() {
+        gson = new Gson();
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
I guess this is a copy paste leftover. So just delete the method.

Reply:
Done